proxy/http/HttpConfig.cc

```diff
@@ -1255,6 +1257,7 @@ HttpConfig::reconfigure()
   params->oride.parent_connect_attempts       = m_master.oride.parent_connect_attempts;
   params->per_parent_connect_attempts         = m_master.per_parent_connect_attempts;
   params->parent_connect_timeout              = m_master.parent_connect_timeout;
+  params->parent_failures_update_hostdb       = m_master.parent_failures_update_hostdb;
```

Comment:
Thats a good point.  In our case we would never mark a parent down in hostdb so completely off works for us.  However making it overidable is a good idea.  Let me make changes.